src/test/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/adjourncase/AdjournCasePreviewServiceTest.java

```diff
@@ -1742,6 +1756,192 @@ public void givenCaseWithFirstWithInvalidDateTime_thenDisplayAnErrorAndDoNotDisp
+        sscsCaseData.setAdjournCaseTypeOfNextHearing("faceToFace");
+        sscsCaseData.setAdjournCaseNextHearingDateType("firstAvailableDateAfter");
+        sscsCaseData.setAdjournCaseNextHearingFirstAvailableDateAfterDate("2020-01-01");
+        sscsCaseData.setAdjournCaseTime(AdjournCaseTime.builder().build());
+
+        final PreSubmitCallbackResponse<SscsCaseData> response = service.preview(callback, DocumentType.DRAFT_ADJOURNMENT_NOTICE, USER_AUTHORISATION, false);
+
+        String error = response.getErrors().stream().findFirst().orElse("");
+        assertEquals("An interpreter is required but no language is set", error);
+        assertNull(response.getData().getAdjournCasePreviewDocument());
+    }
+
+    @Test
+    public void givenInterpreterLanguageRequiredAndProvided_thenErrorShouldNotBeInterpreterLanguageNotSet() {
+
+        sscsCaseData.setAdjournCaseGenerateNotice("yes");
+        sscsCaseData.setAdjournCaseInterpreterRequired("yes");
+        DynamicListItem item = new DynamicListItem("signPalantypist", "Sign Palantypist/Speech to TX");
+        sscsCaseData.setAdjournCaseInterpreterLanguage(new DynamicList(item, Arrays.asList()));
+        sscsCaseData.setAdjournCaseTypeOfNextHearing("faceToFace");
+        sscsCaseData.setAdjournCaseNextHearingDateType("firstAvailableDateAfter");
+        sscsCaseData.setAdjournCaseNextHearingFirstAvailableDateAfterDate("2020-01-01");
+        sscsCaseData.setAdjournCaseTime(AdjournCaseTime.builder().build());
+
+        final PreSubmitCallbackResponse<SscsCaseData> response = service.preview(callback, DocumentType.DRAFT_ADJOURNMENT_NOTICE, USER_AUTHORISATION, false);
+
+        String error = response.getErrors().stream().findFirst().orElse("");
+        assertNotEquals("An interpreter is required but no language is set", error);
+        assertNull(response.getData().getAdjournCasePreviewDocument());
+    }
+
+    @Test
+    public void givenASignPalantypistKey_ThenExtractTheInterpreterDescription(){
+        commonSetupForInterpreterDescription(sscsCaseData);
+        DynamicListItem item = new DynamicListItem("signPalantypist", "Sign Palantypist/Speech to TX");
+        sscsCaseData.setAdjournCaseInterpreterLanguage(new DynamicList(item, Arrays.asList()));
+        final PreSubmitCallbackResponse<SscsCaseData> response = service.preview(callback, DocumentType.DRAFT_ADJOURNMENT_NOTICE, USER_AUTHORISATION, true);
+        NoticeIssuedTemplateBody templateBody = verifyTemplateBody(NoticeIssuedTemplateBody.ENGLISH_IMAGE, "Appellant Lastname", "face to face hearing", true);
+        assertEquals("a sign language interpreter (Sign Palantypist/Speech to TX)", templateBody.getAdjournCaseTemplateBody().getInterpreterDescription());
+    }
+
+    @Test
+    public void givenASignLanguageOthersKey_ThenExtractTheInterpreterDescription(){
+        commonSetupForInterpreterDescription(sscsCaseData);
+        DynamicListItem item = new DynamicListItem("signLanguage (Others", "Others");
```

Comment:
Please use values from the real reference data
https://github.com/hmcts/sscs-common/blob/master/src/main/resources/reference-data/sign-languages.json